### PR TITLE
Fix Call example on localnet

### PR DESCRIPTION
**Pull request by fadeev**
## Summary by CodeRabbit

- **New Features**
	- Updated tutorials for creating contracts on ZetaChain, including accurate deployment commands and contract addresses.
	- Enhanced instructions for invoking the `Connected` contract, with improved examples for withdrawing tokens and making calls.
	- Introduced a new public `swap` function in the `SwapToAnyToken` contract for direct token swaps on ZetaChain.
	- Added flexibility to the `SwapToAnyToken` contract, allowing users to choose between withdrawing swapped tokens or retaining them on ZetaChain.

- **Bug Fixes**
	- Corrected typos in deployment commands and updated contract addresses for clarity.
	- Clarified handling of gas fees and token approvals in the swap contract tutorial.

### CODE REVIEW

**Comment by fadeev** [comment]
@bbbeeeee @brewmaster012 @zeta-chain/fullstack please, review.

File: src/pages/developers/tutorials/swap-any.mdx [hunk 1]
```diff
@@ -270,7 +270,7 @@ Once your environment is set up, compile the contract and deploy it to localnet
 using the following command:
 
 ```
-yarn deploy-any:localnet
+npx hardhat deploy --name SwapToAnyToken --network localhost --gateway 0x5FC8d32690cc91D4c39d9d3abcBD16989F875707 --uniswap-router 0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0
```
Comment (hernan-clich) [inline src/pages/developers/tutorials/swap-any.mdx:273]: Can we use the same syntax here as in the other commands that were updated below? I believe it works better in terms of UX, this one forces the user to scroll horizontally, which is not a great experience.

i.e.:
```bash
npx hardhat swap-from-evm \
  --network localhost \
  --receiver 0xc351628EB244ec633d5f21fBD6621e1a683B1181 \
  --amount 1 \
  --target 0x05BA149A7bd6dC1F937fA9046A9e05C05f3b18b0 \
  --recipient 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 \
  --withdraw true
```

File: src/pages/developers/tutorials/swap.mdx [hunk 2]
```diff
@@ -272,7 +272,7 @@ their native chain, whether it's an EVM chain or Bitcoin.
 
 ```
 npx hardhat compile --force
-npx hardhat deploy --gateway 0x6c533f7fe93fae114d0954697069df33c9b74fd7 --network zeta_testnet
+npx hardhat deploy --gateway 0x6c533f7fe93fae114d0954697069df33c9b74fd7 --uniswap-router 0x2ca7d64A7EFE2D62A725E2B35Cf7230D6677FfEe --network zeta_testnet
```
Comment (hernan-clich) [inline src/pages/developers/tutorials/swap.mdx:275]: Same about the command syntax, this one is also too long.

**Review by hernan-clich — APPROVED** [review]
Tested ACK

**Pull request merged**